providers/novu-email/src/lib/novuEmail.provider.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  IEmailOptions,
+  ISendMessageSuccessResponse,
+  ICheckIntegrationResponse,
+  CheckIntegrationResponseEnum,
+} from '@novu/stateless';
+
+import { SendgridEmailProvider } from '@novu/sendgrid';
+
+export class NovuEmailProvider extends SendgridEmailProvider {
```

Comment:
this was one of the things I dislike this package is useless to anyone else except Novus cloud version... My suggestion is putting this inside of the api server...

Reply:
If we keep it outside providers package then again we need to write custom logic, if we keep it here... , even though many guys not using it is like any other provider. already we have many providers and many guys they do not use many providers and they use which ever providers they need/interested... Lest assume this provider is for cloud and end user is Novu cloud